openfisca_us/parameters/hhs/ccdf/age_limit.yaml

```
@@ -0,0 +1,10 @@
+description: Age limit for CCDF by state
```

To clarify `<` vs `<=`
```suggestion
description: Age at which children are no longer eligible for CCDF
```